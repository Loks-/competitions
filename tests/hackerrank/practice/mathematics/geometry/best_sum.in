11
2
-1 5
4 -5
1
-2
0
3
8 -1 2
4 -1 -5
4
1 7 1 0
-5 3 -5 -6
1
0
6
4
0 -8 -2 0
-8 8 8 0
3
-2 6 -5
-6 -5 -3
4
0 5 1 -8
-3 -3 2 -5
3
5 -7 -2
3 2 -2
5
4 7 -5 1 -3
-7 1 6 6 9
5
-4 3 -9 -2 4
2 -3 -8 -7 0
