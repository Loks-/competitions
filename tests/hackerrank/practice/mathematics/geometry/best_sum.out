50
4
101
260
36
356
197
130
81
490
400
